src/test/java/uk/gov/ons/ctp/response/sample/endpoint/CsvIngesterSocialTest.java

```diff
@@ -47,39 +51,90 @@
   @Mock
   private SampleService sampleService;
 
+  @Mock
+  private SampleAttributesRepository sampleAttributesRepository;
+
   @Rule
   public ExpectedException thrown = ExpectedException.none();
 
-  /**
-   * take a named test file and create a copy of it - is because the ingester
-   * will delete the source csv file after ingest
-   *
-   * @param fileName source file name
-   * @return the newly created file
-   * @throws Exception oops
-   */
-  private MockMultipartFile getTestFile(String fileName) throws Exception {
-    Path csvFileLocation = Paths.get(getClass().getClassLoader().getResource("csv/" + fileName).toURI());
-    MockMultipartFile multipartFile = new MockMultipartFile("file", fileName, "csv",
-        Files.readAllBytes(csvFileLocation));
-    return multipartFile;
-  }
+  @Captor
+  public ArgumentCaptor<List<SocialSampleUnit>> argumentCaptor;
+
 
   @Test
-  public void testBlueSky() throws Exception {
+  public void testIngestSocialSampleFile() throws Exception {
+    // Given
     SampleSummary newSummary = new SampleSummary();
-    csvIngester.ingest(newSummary, getTestFile("social-survey-sample.csv"));
-    verify(sampleService, times(1)).processSampleSummary(eq(newSummary),
-        anyListOf(SocialSampleUnit.class));
+    String csv = "Prem1,Prem2,Prem3,Prem4,District,PostTown,Postcode,CountryCode\n" +
```

Comment:
Should this be in a csv file?

Reply:
I think it makes the test more readable if you have the data with it. Saves switching to another file to understand what the test data is. If it was a few lines of data I'd agree